EleksTubeHAX_pio/src/main.cpp

```diff
@@ -495,7 +498,7 @@ void HandleGesture() {
 void setupMenu() {
   tfts.chip_select.setHoursTens();
   tfts.setTextColor(TFT_WHITE, TFT_BLACK);
-  tfts.fillRect(0, 120, 135, 120, TFT_BLACK);
```

Comment:
Why was this necessary?

Reply:
Good question :) This change is not needed at all. Actually I don't know, when and why this change was made initially. I just not noticed it and copied and pasted it with the mergening tool.